src/main/kotlin/pl/edu/agh/gem/external/dto/report/ReportNotificationRequest.kt

```diff
@@ -0,0 +1,8 @@
+package pl.edu.agh.gem.external.dto.report
+
+data class ReportNotificationRequest(
+    val id: String,
```

Comment:
think we dont have to send this. Email Sender doesnt need this field

Reply:
i was thinking to add some kind of check to not send the same email twice, in case of timeout or retires of reportJob